### PR TITLE
permit new error message (R-devel)

### PR DESCRIPTION
**Pull request by AshesITR**
Currently, CI builds on r-devel are failing because the parser error message for input `\` changed from "unexpected input" to "unexpected end of line".

### CODE REVIEW

**Comment by MichaelChirico** [comment]
more robust would be to generate the error minimally with base R code only

File: tests/testthat/test-error.R [hunk 1]
```diff
@@ -39,10 +39,25 @@ test_that("returns the correct linting", {
     equals_na_linter
   )
 
+  # Error message changed in R-devel as of 2020/12
+  old_lang <- Sys.getenv("LANG")
+  Sys.setenv(LANG = "en")
+  on.exit(Sys.setenv(LANG = old_lang), add = TRUE)
```
Comment (MichaelChirico) [inline tests/testthat/test-error.R:45]: another step further :)

File: tests/testthat/test-error.R [hunk 2]
```diff
@@ -39,10 +39,25 @@ test_that("returns the correct linting", {
     equals_na_linter
   )
 
+  # Error message changed in R-devel as of 2020/12
+  old_lang <- Sys.getenv("LANG")
+  Sys.setenv(LANG = "en")
```
Comment (MichaelChirico) [inline tests/testthat/test-error.R:44]: I believe the variable is `LANGUAGE` btw

Comment (AshesITR) [reply tests/testthat/test-error.R]: Thanks, had a brain fart :D
Tests were failing locally with LANG.

**Review by MichaelChirico — APPROVED** [review]

**Pull request merged**